sdk/liftrqumulo/Azure.ResourceManager.Qumulo/tests/Azure.ResourceManager.LiftrQumulo.Tests.csproj

```diff
@@ -0,0 +1,5 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Test project should match the name / namespace of the src project.

Reply:
changed the namespace to `Qumulo`